picky/Cargo.toml

```diff
@@ -75,7 +75,7 @@ rc2 = { version = "0.8", optional = true }
 pbkdf2 = { version = "0.12", optional = true }
 hmac = { version = "0.12", optional = true }
 
-ucs2 = { version = "0.3", optional = true }
+ucs2 = "0.3"
```

Comment:
Should we keep this optional, maybe as a default feature?

Reply:
Idk, ucs2 is actually very small crate, around ~200 LoC. It is used by pkcs12 feature internally (for password formatting) but we could keep it as optional feature, I actually don't have a strong opinion on that, I think we could keep it non-optional as it may be useful for other things in picky (such as certificate building), but it is ok if we will keep it optional. What are your thoughts on that?

Reply:
Fair enough, let’s keep it!